packages/ui/src/i18n/dictionaries/authenticator/de.ts

```diff
@@ -1,68 +1,86 @@
-export const deDict = {
+import { AuthenticatorDictionary } from './types';
+
+export const deDict: AuthenticatorDictionary = {
   'Account recovery requires verified contact information':
     'Zurücksetzen des Account benötigt einen verifizierten Account',
-  'An account with the given email already exists.':
-    'Ein Account mit dieser Email existiert bereits.',
   'Back to Sign In': 'Zurück zur Anmeldung',
   'Change Password': 'Passwort ändern',
+  Changing: 'Ändern von',
   Code: 'Code',
-  Confirm: 'Bestätigen',
   'Confirm Password': 'Passwort bestätigen',
-  'Confirm a Code': 'Code bestätigen',
-  'Confirm Sign In': 'Anmeldung bestätigen',
   'Confirm Sign Up': 'Registrierung bestätigen',
   'Confirm SMS Code': 'SMS-Code bestätigen',
   'Confirm TOTP Code': 'TOTP-Code bestätigen',
+  Confirm: 'Bestätigen',
   'Confirmation Code': 'Bestätigungs-Code',
+  Confirming: 'Wird bestätigt',
   'Create a new account': 'Einen neuen Account erstellen',
-  'Create account': 'Hier registrieren',
   'Create Account': 'Account erstellen',
   'Creating Account': 'Account wird erstellt',
   Email: 'Email',
   'Enter your code': 'Geben Sie Ihren Code ein',
-  'Enter your password': 'Geben Sie Ihr Passwort ein',
-  'Enter your username': 'Geben Sie Ihren Benutzernamen ein',
-  'Enter your phone number': 'Geben Sie Ihre Telefonnummer ein',
   'Enter your email': 'Geben Sie Ihre Email ein',
+  'Enter your phone number': 'Geben Sie Ihre Telefonnummer ein',
+  'Enter your username': 'Geben Sie Ihren Benutzernamen ein',
   'Forgot your password?': 'Passwort vergessen? ',
-  'Forgot Password': 'Passwort vergessen',
   'Hide password': 'Passwort verbergen',
+  Loading: 'Wird geladen',
+  'New password': 'Neues Passwort',
+  Password: 'Passwort',
+  'Phone Number': 'Telefonnummer',
+  'Resend Code': 'Code erneut senden',
+  'Reset your Password': 'Zurücksetzen des Passwort',
```

Comment:
These did not change any of the user submitted translations correct?

Reply:
Correct